engine/classes/ElggMemcache.php

```diff
@@ -107,6 +107,9 @@ public function __construct($namespace = 'default') {
 		if (isset($this->CONFIG->memcache_expires)) {
 			$this->expires = $this->CONFIG->memcache_expires;
 		}
+		
+		// make sure memcache is reset
+		_elgg_services()->events->registerHandler('cache:flush', 'system', array($this, 'clear'));
```

Comment:
for ElggMemcache it's clear for Memcache it's actualy flush. for simplecache it's invalidate, for system_cache it's reset.....

so let's just stick with flush (as that is the action it was envisioned for)